### PR TITLE
newsletter 2019-08-15

### PR DESCRIPTION
**Pull request by ryan-shea**

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -890,22 +890,22 @@ def render_new_bounty_roundup(to_email):
...
+"Are we done with hackathons?" you may ask. Absolutely not. Coming up after ETHBerlin is our next virtual hackathon, Ethereal Blocks. This hackathon has $50k of prizes from Portis, Ethereum, Matic, Status, and more. Place your lego blocks towards a more open, transparent internet: and if you win, you'll get the chance to present at the Ethereal Summit on 9/15. We'll be dropping more details in the coming weeks. Follow us on <a href="https://twitter.com/gitcoin">Twitter</a> and bookmark the <a href="https://hackathons.gitcoin.co/ethereal-blocks/">hackathon homepage.</a>  
 </p>
 <p>
-Last, if you're looking for something to watch this weekend, we're rolling back the tapes on our livestream with last week's Grow Ethereum focused livestream. Check it on <a href="https://www.youtube.com/gitcoinmedia">Gitcoin Media.</a>
+We're rearing up for the next round of Gitcoin Grants CLR matching. If you're a project in need of funding, or are a supporter of open source, be sure to peruse the Gitcoin Grants homepage <a href="https://gitcoin.co/grants/">- located here -</a> to find projects that interest you or post your own. Expect more information as we get further along in Q3 of 2019!
```
Comment (thelostone-mc) [inline app/retail/emails.py]: *gearing up  ?

Comment (PixelantDesign) [reply app/retail/emails.py]: I think gearing up makes more sense @thelostone-mc

**Comment by thelostone-mc** [comment]
Testing it locally looks solid

**Comment by thelostone-mc** [comment]
UPDATE: sent out newsletter

**Pull request merged**